kong/examples/kube/README.md

```diff
@@ -0,0 +1,82 @@
+---
+# Ingest metrics using the Kong integration
+
+The OTel Collector has a variety of [third party receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver) that provide integration with a wide variety of metric sources.
+
+Please note that not all metrics receivers available for the OpenTelemetry Collector have been tested by Lightstep Observability, and there may be bugs or unexpected issues in using these contributed receivers with Lightstep Observability metrics. File any issues with the appropriate OpenTelemetry community.
+{: .callout}
+
+## Prerequisites for local installation
+
+You must have a Lightstep Observability [access token](/docs/create-and-manage-access-tokens) for the project to report metrics to.
+
+## Running the Example
+
+### Run the example by running:
+```bash
+make all
+```
+that will create local kubernetes cluster and install kong and collector.
+
+### Otherwise you can use following commands:
+
+#### 1. Create local kind cluster
+
+```bash
+kind create cluster --config kind-config.yaml
+```
+
+#### 2. Run kong
+
+Need to create kong namespace, add kong secret(empty test secret in this case), install kong and expose metrics endpoint
```

Comment:
IMO this sounds a little better: s/Need to/Next,

Reply:
done